packages/calcite-components/src/components/input-number/input-number.e2e.ts

```diff
@@ -1552,44 +1552,31 @@ describe("calcite-input-number", () => {
     await typeNumberValue(page, "7000");
     await page.waitForChanges();
     expect(await calciteInput.getProperty("value")).toBe("7000");
-    expect(await input.getProperty("value")).toBe("7000");
+    expect(await input.getProperty("value")).toBe("7,000");
 
     await page.keyboard.press("ArrowLeft");
     await page.keyboard.press("ArrowLeft");
     await page.keyboard.press("ArrowLeft");
     await page.keyboard.press("Backspace");
     await page.waitForChanges();
-    expect(await calciteInput.getProperty("value")).toBe("0");
-    expect(await input.getProperty("value")).toBe("000");
+    expect(await calciteInput.getProperty("value")).toBe("7000");
+    expect(await input.getProperty("value")).toBe("7000");
```

Comment:
Based on the modified expected value here, it looks like you are deleting the group separator instead of the `7`. You will need to add another `ArrowLeft` keypress now that there is an additional character.

Alternatively you can do this, which doesn't depend on the length of the number:

```ts
    await page.keyboard.press("Home");
    await page.keyboard.press("Delete");
```

The other expected values below will also need to be updated to ensure they are testing the leading zero fix.